internal/functions/serve/serve.go

```diff
@@ -155,6 +156,12 @@ EOF
 			Env:          append(env, userEnv...),
 			Entrypoint:   entrypoint,
 			ExposedPorts: nat.PortSet{"8081/tcp": {}},
+			Healthcheck: &container.HealthConfig{
+				Test:     []string{"CMD", "bash", "-c", "printf \\0 > /dev/tcp/localhost/8081"},
+				Interval: 2 * time.Second,
+				Timeout:  2 * time.Second,
+				Retries:  10,
```

Comment:
Let's use the default healthcheck params, in case they're consuming a lot of resources: https://github.com/supabase/cli/issues/1367

Though it's more applicable to `db` or `postges-meta` which have costlier checks

Reply:
There's no default health check defined for edge runtime container.

```bash
qiao@Qiaos-MacBook-Pro cli % docker ps -a
CONTAINER ID   IMAGE                            COMMAND                  CREATED          STATUS                    PORTS                                              NAMES
484b18c6f575   supabase/edge-runtime:v1.8.1     "sh -c 'mkdir -p /ho…"   48 minutes ago   Up 48 minutes             8081/tcp                                           supabase_edge_runtime_cli
```

Reply:
Anyway I've responded to the other ticket. I don't think the fix is as straightforward as increasing the interval universally to a high value.

Reply:
TIL, didn't know the flag was a recent addition.